src/components/Accordion/AccordionSummary/AccordionSummary.scss

```diff
@@ -0,0 +1,43 @@
+@use '../../variables';
+@use '../../../../styles/mixins';
+
+$block: '.#{variables.$ns}accordion-summary';
+
+@mixin accordion-item-trigger-header($size) {
+    #{$block}__trigger {
```

Comment:
Why wrap another selector with a class in a mixin? This makes it difficult to read the styles. In a good way, this should be described in the `&__trigger` block